devdocs/settingUp.md

```diff
@@ -115,13 +115,18 @@ Refer to [this document](staticAnalysis.md) for details on the tools used, how t
 
 
 
-1. TEAMMATES automated testing requires Firefox (works on Windows and OS-X).
-   Only Firefox between versions 38.0.5 and 46.0 are supported, although the primary support is for 46.0.
+1. TEAMMATES automated testing requires Firefox (versions 38.0.5 to 46.0.1) or Chrome, both of which work on Windows and Mac OS.
+   It is recommended to use Firefox 46.0 as this is the browser used in Travis build.
    To downgrade your Firefox version, obtain the executable from [here](https://ftp.mozilla.org/pub/mozilla.org/firefox/releases/).
    If you want to use a different path for this version, choose `custom setup` during install.
    After installation, specify the Firefox path in `test.properties` by modifying the `test.firefox.path` property.
    Remember to disable the auto-updates (`Options → Advanced tab → Update`).
    
+   If you want to use Chrome for testing, you need to
+   download the latest stable chromedriver from [here](https://sites.google.com/a/chromium.org/chromedriver/downloads)
+   and update the `test.chromedriver.path` value in `test.properties`.
+   The site will also inform the versions of Chrome that can be used with the driver.
```

Comment:
By the way, the latest Chrome driver is said to support only up to v52, but I've tested with Chrome 53 and is fine.